### PR TITLE
OPHJOD-280: Add JOD theme and addons

### PR DESCRIPTION
**Pull request by sauanto**
## Description
- Add JOD theme and addons

## Related JIRA ticket

https://jira.eduuni.fi/browse/OPHJOD-280

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ['./lib/**/*.{js,ts,jsx,tsx}'],
+  darkMode: 'selector',
   theme: {
+    screens: {
+      lg: '1440px',
+    },
```
Comment (ketsappi) [inline tailwind.config.js:8]: Not so familiar with Tailwind, so why there is need to specify just one screen size and specifially this?

File: .storybook/preview.ts [hunk 2]
```diff
@@ -9,7 +11,29 @@ const preview: Preview = {
...
+      },
+    },
   },
+  decorators: [
+    withThemeByClassName<ReactRenderer>({
+      themes: {
+        light: '',
+        dark: 'dark',
+      },
+      defaultTheme: 'light',
```
Comment (ketsappi) [inline .storybook/preview.ts:34]: Why light theme's value is empty and not `light`?

File: tailwind.config.js (hunk 1 above)
Comment (sauanto) [reply tailwind.config.js:8]: We should have breakpoints at least for desktop and mobile. Current `1440px ` is from design and it's the only one we know of so far. See default breakpoints https://tailwindcss.com/docs/screens.

File: .storybook/preview.ts (hunk 2 above)
Comment (sauanto) [reply .storybook/preview.ts:34]: A good point. I add it.

**Pull request merged**